docs/src/pages/api-reference/react.mdx

```diff
@@ -164,6 +161,13 @@ export function MultiUploader() {
     },
   });
 
+  const fileTypes = permittedFileInfo?.config ? Object.keys(permittedFileInfo?.config) : [];
+
+  const { getRootProps, getInputProps } = useDropzone({
+    onDrop,
+    accept: fileTypes ? generateClientDropzoneAccept(fileTypes) : undefined,
```

Comment:
note for later: We shoudl make this easier/less steps when we control the useDropzone